sql/moz-fx-data-shared-prod/amo_prod/desktop_addons_by_client_v1/query.sql

```diff
@@ -52,9 +64,13 @@ SELECT
   ARRAY(
     SELECT AS STRUCT
       addon.id,
-      udf.mode_last(array_agg(addon.version)) AS version,
+      -- Same methodology as for app_version above.
+      ARRAY_AGG(addon.version ORDER BY mozfun.norm.truncate_version(addon.version, "minor") DESC)[
+        SAFE_OFFSET(0)
+      ] AS version,
     FROM
       UNNEST(addons) AS addon
+      WITH OFFSET AS i
```

Comment:
`i` seems to be unused

Reply:
Will remove. This is a holdover from a previous variant of the design.